engine/api/src/main/scala/pl/touk/nussknacker/engine/api/ContextInterpreter.scala

```diff
@@ -0,0 +1,9 @@
+package pl.touk.nussknacker.engine.api
+
+object ContextInterpreter {
```

Comment:
Maybe better name would be `VariableConstants`?

Reply:
Fixed. Renamed to `VariableConstants`.